app/src/androidTest/java/com/droidhats/campuscompass/views/FloorFragmentTest.kt

```diff
@@ -0,0 +1,154 @@
+package com.droidhats.campuscompass.views
+
+import androidx.navigation.testing.TestNavHostController
+import androidx.test.core.app.ApplicationProvider
+import androidx.test.espresso.Espresso.onView
+import androidx.test.espresso.action.*
```

Comment:
Change * import to specify specific imports

Reply:
Fixed